commandline/WSL/install_guide.md

```diff
@@ -1,84 +1,93 @@
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu, windows 10
+author: scooley
+ms.date: 7/11/2017
 ms.topic: article
 ms.prod: windows-subsystem-for-linux
 ms.service: windows-subsystem-for-linux
 ms.assetid: 7afaeacf-435a-4e58-bff0-a9f0d75b8a51
 ---
 
-# Installation Guide
+# Windows 10 Installation Guide
 
-> **Important note**  
-	This is the first release of Bash on Windows and it is branded "beta" deliberately - it's not yet complete! You should expect many things to work and for some things to fail! We greatly appreciate you using Bash on Windows and helping us identify the issues we need to fix in order to deliver a great experience.
+## Prerequisites
+Your PC must be running a 64-bit version of Windows 10.
 
-##Prerequisites
-1. Your PC must be running (at a minimum) a 64-bit version of Windows 10 Anniversary Update. **Creators Update is recommended.**
+To find your PC's architecture and Windows build number, open  
+**Settings** > **System** > **About**
 
-> To find your PC's CPU architecture and Windows version/build number, open 
-> **Settings**>**System**>**About**. 
-> Look for the **OS Build** and **System Type** fields.  
+Look for the **OS Build** and **System Type** fields.  
+    ![](media/system.png) 
 
-![](media/system.png) 
+## Install the Windows Subsystem for Linux
+Enable the "Windows Subsystem for Linux" optional feature and reboot.
 
-If your build is below 14393, try checking for updates.
+1. Open PowerShell as Administrator and run:
+    ``` PowerShell
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+    ```
 
+2. Restart your computer when prompted.
 
-## Installation
-In order to run Bash on Windows, you will need to manually:
-1. Turn-on Developer Mode
-2. Enable the "Windows Subsystem for Linux (beta)" feature via the GUI or the command-line:
+## For Windows Insiders -- Install Linux distribution of choice
 
-### Turn on Developer Mode
-1. Open Settings -> Update and Security -> For developers
-2. Select the Developer Mode radio button  
-  ![](media/updateAndSecurity.png)
+> This section is for Windows Insiders (build 16215 or later).  For earlier versions of Windows 10, follow [these instructions using lxrun](install_guide.md#Install-using-lxrun-Anniversary-Update-and-Creators-Update).
 
-### Enable the Windows Subsystem for Linux feature (GUI)
-1. From Start, search for "Turn Windows features on or off" (type 'turn')
-2. Select **Windows Subsystem for Linux (beta)**  
-  ![](media/windowsFeatures.png)
-  
-3. Click OK
+1. Open the Windows Store and choose your favorite Linux distribution.      
+    Here are links directly to the store installers:
+    * [Ubuntu](https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6)
+    * [OpenSUSE](https://www.microsoft.com/store/apps/9njvjts82tjx)
+    * [SLES](https://www.microsoft.com/store/apps/9p32mwbh6cns)

```

Comment:
OpenSUSE and SLES are still not available at the store. It's likely that this doc will get published before they go online. Should we hold off to publishing these links until they go online?